tests/acceptance/features/webUITrashbin/trashbinRestore.feature

```diff
@@ -97,3 +97,22 @@ Feature: Restore deleted files/folders
     And file "lorem-big.txt" should be listed on the webUI
     And file "data.zip" should be listed on the webUI
     And folder "simple-folder" should be listed on the webUI
+
+  @issue-1753
+  Scenario: Restore a file from trashbin whose parent folder is renamed
+    Given the user has created file "simple-folder/file-to-delete-and-restore"
+    And the following files have been deleted by user "user1"
+      | name                                     |
+      | simple-folder/file-to-delete-and-restore |
+    And user "user1" has renamed folder "simple-folder" to "simple-folder-renamed"
+    When the user browses to the trashbin page
+    #And the user restores file "simple-folder/file-to-delete-and-restore" from the trashbin using the webUI
+    And the user tries to restore file "simple-folder/file-to-delete-and-restore" from the trashbin using the webUI
+    Then an error with message "Restoration of file-to-delete-and-restore failed" should be displayed
+    #Then a success message "" should be displayed
```

Comment:
This step is a bit confusing. Please add correct error message or remove the step.

Reply:
done